src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -2650,6 +2768,75 @@ namespace OutputReportPredefined {
         pdchLeedSchStPt11amWedCnt = newPreDefColumn(pdstLeedSchedSetPts, "Days with Same 11am Value");
         pdchLeedSchStPt11pmWednesday = newPreDefColumn(pdstLeedSchedSetPts, "11pm First Wednesday [C]");
         pdchLeedSchStPt11pmWedCnt = newPreDefColumn(pdstLeedSchedSetPts, "Days with Same 11pm Value");
+
+        pdrThermalResilience = newPreDefReport("ThermalResilienceSummary", "ThermR", "Annual Thermal Resilience Summary");
+
+        pdstHIHours = newPreDefSubTable(pdrThermalResilience, "Heat Index Hours");
+        pdchHIHourSafe = newPreDefColumn(pdstHIHours, "Safe (≤ 26.7°C) [Hours]");
+        pdchHIHourCaution = newPreDefColumn(pdstHIHours, "Caution (26.7, 32.2°C] [Hours]");
+        pdchHIHourExtremeCaution = newPreDefColumn(pdstHIHours, "Extreme Caution (32.2, 39.4°C] [Hours]");
+        pdchHIHourDanger = newPreDefColumn(pdstHIHours, "Danger (39.4, 51.7°C] [Hours]");
+        pdchHIHourExtremeDanger = newPreDefColumn(pdstHIHours, "Extreme Danger (> 51.7°C) [Hours]");
+
+        pdstHIOccuHours = newPreDefSubTable(pdrThermalResilience, "Heat Index OccupantHours");
+        pdchHIOccuHourSafe = newPreDefColumn(pdstHIOccuHours, "Safe (≤ 26.7°C) [OccupantHours]");
+        pdchHIOccuHourCaution = newPreDefColumn(pdstHIOccuHours, "Caution (26.7, 32.2°C] [OccupantHours]");
+        pdchHIOccuHourExtremeCaution = newPreDefColumn(pdstHIOccuHours, "Extreme Caution (32.2, 39.4°C] [OccupantHours]");
+        pdchHIOccuHourDanger = newPreDefColumn(pdstHIOccuHours, "Danger (39.4, 51.7°C] [OccupantHours]");
+        pdchHIOccuHourExtremeDanger = newPreDefColumn(pdstHIOccuHours, "Extreme Danger (> 51.7°C) [OccupantHours]");
```

Comment:
What is difference between Hours and OccupantHours? OccupantHours is a new unit and we try to avoid adding them unless we have a good reason.

Reply:
Occupant * hours. So the unit should be [hrs] I guess. I'll change them to be consistent.